documentation/movies.workspace.adoc

```diff
@@ -702,6 +699,22 @@ MATCH path = (person)-[:ACTED_IN]->(m)<-[:DIRECTED]-(d)
 RETURN path;
 ----
 
+To aid query performance, you can add indexes to your data.
+(The dataset you create in this guide is very small, but it is good practice to use indexes as they can greatly improve query performance when the datasets are larger.)
+
+In this small dataset, you are mainly interested in names of people and titles of movies.
+Run the following to add indexes on these properties
+
+[source,cypher]
+----
+CREATE INDEX person_name FOR (p:Person) ON (p.name);
+CREATE INDEX movie_title FOR (m:Movie) ON (m.title)
+----
+
+Note that you can also add constraints to your data to ensure uniqueness for example.
+Adding constraints automatically adds a correpsonding index.
+See link:https://neo4j.com/docs/cypher-manual/current/constraints/[Cypher Manual -> Constraints] for more information.
```

Comment:
Do you need to add a `^` to the hyperlink to open in a new window?

e.g.

```
https://neo4j.com/docs/cypher-manual/current/constraints/[Cypher Manual -> Constraints^]
```

Reply:
Yes, you're right! I'll amend! Thanks!